doc/release/1.12.0-notes.rst

```diff
@@ -6,6 +6,9 @@ This release supports Python 2.7 and 3.4 - 3.5.
 Highlights
 ==========
 
+Details of these improvements can be found below.
+
+* Addition of `nancumsum` and `nancumprod` to the set of nanfunctions.
```

Comment:
Better to add these to "new features" with a brief description -- we'll fill out highlights shortly before cutting the release.